src/krait/cli.py

```diff
@@ -54,8 +54,8 @@
 
 help_links = {}
 
-for obj in help_link_objects.values():
-    help_links.update(obj.links)
+for value in help_link_objects.values():
+    help_links.update(value.links)
```

Comment:
don't mind the name change but I'm not sure 'value' is appropriate. I'll give this some more thought and address this in a later change

Reply:
Oops, this was just some repeated variable warning that showed up on my linter due to the variable `obj` already being used in a function declaration.

https://github.com/taliamax/krait/blob/71e6d21719d50dcffd0e764506d0df519c75b70c/src/krait/cli.py#L458

I guess it's just my linter being weird... my bad